lib/output.js

```diff
@@ -409,7 +409,7 @@ function OutputStream(options) {
             print(encoded);
         },
         print_template_string_chars: function(str) {
-            var encoded = encode_string(str, '`');
+            var encoded = encode_string(str, '`').replace(/\${/g, "\\${");
```

Comment:
I don't follow. That line of code is some sort of whitespace handling.

I think the code is okay for now and I can't come up with a scenario in which it does not work. Feel free to move the fix to a more appropriate function later.